src/content/docs/data-apis/manage-data/manage-data-retention.mdx

```diff
@@ -36,20 +36,20 @@ With [our Data Plus option](/docs/accounts/accounts-billing/new-relic-one-pricin
   src={accountsEditRetentionSettings}
 />
 <figcaption>
-  The <DoNotTranslate>**Data retention**</DoNotTranslate> UI is where you can see your data retention periods, and edit data retention. To view explanations of the table column headers, hover over a header. 
+  The <DoNotTranslate>**Data retention**</DoNotTranslate> UI is where you can see your data retention periods, upgrade your retention plan (Max per contract) and edit effective retention. To view explanations of the table column headers, hover over a header. 
 </figcaption>
 
 ## Requirements [#requirements]
 
-Being able to configure data retention requires Pro or Enterprise edition with billing administrator rights. Account characteristics that do not meet this criteria will block some New Relic organizations from adjusting retention on their own. If you're a Pro or Enterprise organization on a billing administrator account, and you can't edit retention, talk to your New Relic account representative.
+Being able to upgrade your data retention plan ((Max per contract) or edit the effective retention values requires Pro or Enterprise. Account characteristics that do not meet this criteria will block some New Relic organizations from adjusting upgrading and editing effective retention values on their own. If you're a Pro or Enterprise and you can't upgrade your retention plan or edit retentionv values, talk to your New Relic account representative.
```

Comment:
@johnathannewrelic I'm hoping we can drop this sentence, since it seems redundant: Account characteristics that do not meet this criteria will block some New Relic organizations from adjusting upgrading and editing effective retention values on their own.

Not only does it seem redundant, it seems to hedge a bit. For example, it says this may block "some" organizations who aren't using Pro or Enterprise from making changes. Is that true, or does the requirement to use Pro or Enterprise apply across the board to all organizations?

Reply:
It looks like we've dropped the "billing administrator account" language. I just wanted to confirm. 

Oh, I see that this is now down in line 51.

Reply:
I completely agree with you. I was trying to retain a lot of what it originally said out of respect for what was originally approved, while only zeroing in on the facts that needed to be corrected.

Your version looks much better and still delivers the correct facts.